src/bot.py

```
@@ -31,23 +33,35 @@
+    get_context_duration = get_context_end_time - get_context_start_time
+    logger.info(f"Got context from Honcho in {get_context_duration:.2f} seconds")
+
     try:
+        completion_start_time = time.perf_counter()
         completion = openai.chat.completions.create(
             model=MODEL_NAME,
             messages=messages,
         )
+        completion_end_time = time.perf_counter()
+        completion_duration = completion_end_time - completion_start_time
+        logger.info(f"Got completion from LLM in {completion_duration:.2f} seconds")
```

Ruff (0.13.3)

Logging statement uses f-string

(G004)